client/src/App.js

```diff
@@ -62,7 +63,7 @@ function App() {
     <MuiThemeProvider theme={theme}>
       <HashRouter>
         {user._id ? (
-          <>
+          <PollsContextProvider>
```

Comment:
Nothing wrong with placing context here, but it looks like we have all-encompassing contexts currently listed in `src/index.js`

I created #106 asking to move the rest to where this is

Reply:
I did this here because the other contexts include the unauthorized routes (Login/Signup).
For the PollsContext I need the user to be logged in to get the polls. There were other ways to do this, but this is the fastest, and the less children I context has, less re-renders. (I would put the friends context here too).